docs/source/releasenotes.rst

```diff
@@ -22,6 +22,8 @@ Release notes
   with the keyword `Set default URL scheme`.
 - Library **RPA.Hubspot**: Fix several bugs and improve logging (:issue:`504`,
   :issue:`505`, :issue:`506`, and :issue:`507`).
+- Library **RPA.Excel.Files** (:pr:`514`): Corrected example formatting errors.
```

Comment:
This can be placed again under the new empty Upcoming as we had a release meanwhile.